.pipelines/pipeline.user.windows.yml

```diff
@@ -12,9 +12,10 @@ versioning:
     - !!defaultcommand
       name: 'Generate Version'
       command: '.scripts/version.cmd'
+# excluding our source code from being scanned for code sign validation
 signing_options:
-  profile: '170'
-  codesign_validation_glob_pattern: 'file|**/agent/agent/build/libs/applicationinsights-agent-*.jar'
+  profile: 'none'
+  codesign_validation_glob_pattern: 'f:regex|\.(jar|pom)$'  # recursively find files ending in .jar, .pom

```

Comment:
is the pattern still needed for profile `none`?

Reply:
i have one of the project admins review it as well.. he offered some feedback.. it's updated.

Reply:
i will testing it without regex pattern.. once i'm done testing this change.